pymedphys/cli/__init__.py

```diff
@@ -63,22 +65,63 @@ def define_parser():
     return parser
 
 
-def pymedphys_cli():
-    parser = define_parser()
+def get_logging_config():
+    try:
+        config = get_config()
+    except FileNotFoundError:
+        return {}
+
+    try:
+        cli_config = config["cli"]
+        logging_config = cli_config["logging"]
+    except KeyError:
+        return {}
+
+    return logging_config
 
-    args, remaining = parser.parse_known_args()
 
-    loglevel = logging.WARNING
-    logformat = "%(asctime)s %(levelname)-8s %(message)s"
+def run_logging_basic_config(args):
+    logging_config = get_logging_config()
 
+    if "level" not in logging_config:
+        logging_config["level"] = logging.WARNING
+    else:
+        try:
+            logging_config["level"] = getattr(logging, logging_config["level"].upper())
+        except AttributeError:
+            pass
```

Comment:
if someone misconfigures the level of logging, e.g. they put in warnning (I intentionally misspelled that... as an example of what someone might do) or status (a logging level available in NLog, Log4J, and other related logging frameworks), then... what happens?
I guess this would default to WARNING because that's what logging defaults to?
It looks like printing (because logging isn't working yet) the AttributeError content would be a kindness, e.g.:

except AttributeError as error:
    print(str(error))

or start adding to a list of errors found during logging configuration build up, and then log them all once the basicConfig is finished, and do so at the error level...

alternatively, I think one could just allow the inappropriate string value to be assigned to the (local variable) logging_config dict and leave it to basicConfig (which I think is a wrapper of dictConfig) to report the error.  But in that scenario, the misconfigured value would not show up if --verbose or --debug was in use on the command line.

Reply:
> if someone misconfigures the level of logging, e.g. they put in warnning (I intentionally misspelled that... as an example of what someone might do) or status (a logging level available in NLog, Log4J, and other related logging frameworks), then... what happens?
> I guess this would default to WARNING because that's what logging defaults to?

It wouldn't default to WARNING it would just leave `logging_config["level"]` as it began (exactly as it appeared in the config file), in the example you gave it would try pass `level='warnning'` to basicConfig, resulting in the following:

```python
In [1]: import logging                                                                    

In [2]: logging.basicConfig(level='warnning')                                             
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-2-954734a85d38> in <module>
----> 1 logging.basicConfig(level='warnning')

~/.pyenv/versions/3.7.8/lib/python3.7/logging/__init__.py in basicConfig(**kwargs)
   1914             level = kwargs.pop("level", None)
   1915             if level is not None:
-> 1916                 root.setLevel(level)
   1917             if kwargs:
   1918                 keys = ', '.join(kwargs.keys())

~/.pyenv/versions/3.7.8/lib/python3.7/logging/__init__.py in setLevel(self, level)
   1351         Set the logging level of this logger.  level must be an int or a str.
   1352         """
-> 1353         self.level = _checkLevel(level)
   1354         self.manager._clear_cache()
   1355 

~/.pyenv/versions/3.7.8/lib/python3.7/logging/__init__.py in _checkLevel(level)
    190     elif str(level) == level:
    191         if level not in _nameToLevel:
--> 192             raise ValueError("Unknown level: %r" % level)
    193         rv = _nameToLevel[level]
    194     else:

ValueError: Unknown level: 'warnning'
```

Which I think is sufficiently informative :).

But, my whole point of that code was to allow users to provide an integer number such as `10` or text value such as `'debug'`, but turns out `basicConfig` already supports that out of the box. So it was superfluous therefore, I removed that piece of code all together.